src/main/java/janggi/domain/Board.java

```diff
@@ -28,30 +43,60 @@ private void validateTeamIsNotNull(final Team team1, final Team team2) {
     }
 
     private void validateCountryIsNotSame(final Team team1, final Team team2) {
-        if(team1.isSameCountry(team2)) {
+        if (team1.isSameCountry(team2)) {
             throw new IllegalArgumentException("두 개의 장기판의 나라는 서로 달라야 합니다.");
         }
     }
 
-    public Map<Position, Piece> getBoard() {
-        final Map<Position, Piece> status = new HashMap<>();
-        status.putAll(team1.getPieces());
-        status.putAll(team2.getPieces());
-        return Collections.unmodifiableMap(status);
+    public Map<Country, List<Piece>> getBoard() {
+        final Map<Country, List<Piece>> board = new HashMap<>();
+        board.put(team1.getCountry(), team1.getPieces());
+        board.put(team2.getCountry(), team2.getPieces());
+        return Collections.unmodifiableMap(board);
     }
 
     public void move(Position fromPosition, Position tagetPosition) {
         if (isTeam1Turn) {
-            team1.move(fromPosition, tagetPosition, team2.getPieces());
+            team1.move(fromPosition, tagetPosition, team2);
             nextTurn();
-        }
-        if (!isTeam1Turn) {
-            team2.move(fromPosition, tagetPosition, team1.getPieces());
+        } else {
+            team2.move(fromPosition, tagetPosition, team1);
             nextTurn();
         }
     }
 
     private void nextTurn() {
         isTeam1Turn = !isTeam1Turn;
     }
+
+    public boolean isEnd() {
+        return team1.isEnd() || team2.isEnd();
+    }
+
+    public Country getCurrentCountry() {
+        return getCurrentTeam().getCountry();
+    }
+
+    private Team getCurrentTeam() {
+        if (isTeam1Turn) return team1;
```

Comment:
```suggestion
        if (isTeam1Turn) {
            return team1;
        }
```

표준 컨벤션에 따라 if문에서 중괄호는 필수로 사용해주세요 😄 
* 참고 : https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

Reply:
해당 내용 전반적으로 체크해주세요~

Reply:
이런 컨벤션도 있었네요! 감사합니다.

[style: if문에 중괄호를 사용하지 않았던 것을 컨벤션에 맞게 수정](https://github.com/woowacourse/java-janggi/pull/86/commits/688fccece9846eb97edbac9343e83e1001681411)